src/Channel.h

```diff
@@ -68,6 +68,8 @@ class Channel : public QObject {
 
 		bool bInheritACL;
 
+		unsigned int uiMaxUsers;
```

Comment:
Please add a doxygen comment stating (very briefly) the purpose of the variable and - most importantly - the special meaning of 0.